scene/gui/view_panner.h

```diff
@@ -45,7 +45,17 @@ class ViewPanner : public RefCounted {
 		SCROLL_PANS,
 	};
 
+	enum PanAxis {
+		PAN_AXIS_BOTH,
+		PAN_AXIS_HORIZONTAL,
+		PAN_AXIS_VERTICAL,
+	};
+
 private:
+	int scroll_speed = 32;
```

Comment:
Shouldn't this be float?
Not like it matters though.

Reply:
I don't know, as it's basically a "number of pixel" per scroll event, I thought I made sense to make it an integer. However, it always gets multiplied by a Vector2 (thus floats), so it could be changed too.